gamepanel/game.php

```diff
@@ -194,6 +194,72 @@ function gameIcons()
 		return $buf;
 	}
 
+	/*

```

Comment:
We'll want to remove any commented out code that isn't needed for the new functionality.

Reply:
Done